rogue-thi-app/pages/api/cl-events.js

```diff
@@ -161,19 +161,27 @@ export async function getAllEventDetails(username, password) {
 
   await login(fetch, username, password)
 
-  const remoteEvents = []
-  for (const url of await getEventList(fetch)) {
+  const eventUrls = await getEventList(fetch)
+  const eventPromises = eventUrls.map(async (url) => {
```

Comment:
This was a synchronous loop intentionally so that we don't spam the server with 30 requests at the same time

Reply:
This is the kind of thing where maybe we should have documented why something was done a certain way 😅